LICENSE

```diff
@@ -1,6 +1,39 @@
-## This is the master copy of the PEcAn License
```

Comment:
Kept both licenses in this file, just swapped BSD3 to the top now that we're past the halfway point